exchanges/subscription/list_test.go

```diff
@@ -91,3 +91,12 @@ func TestAssetPairs(t *testing.T) {
 		assert.ErrorIs(t, err, expErr, "Should error correctly on GetPairFormat")
 	}
 }
+
+func TestListClone(t *testing.T) {
+	t.Parallel()
+	l := List{{Channel: TickerChannel}, {Channel: OrderbookChannel}}
+	n := l.Clone()
+	assert.NotSame(t, n, l, "Slices must not be the same")
+	require.NotEmpty(t, n, "List must not be empty")
+	assert.NotSame(t, n[0], l[0], "Subscriptions must be cloned")
```

Comment:
I think it would be worthwhile to prove in some capacity that you are actually cloning things, that something is equal

Reply:
Fixed aace80f